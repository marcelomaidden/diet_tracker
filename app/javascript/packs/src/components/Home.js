import React from 'react';

const Home = () => (
  <div>
    <h1>Home page</h1>
  </div>
);

export default Home;
