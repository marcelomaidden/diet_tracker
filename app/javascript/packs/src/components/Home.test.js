import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';

import Home from './Home';

let container = null;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('should render a Home component', () => {
  act(() => {
    render(
      <Home />,
      container,
    );
  });

  expect(
    container.innerHTML,
  ).toMatchSnapshot();
});