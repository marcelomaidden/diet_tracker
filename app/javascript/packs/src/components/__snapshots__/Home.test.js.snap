// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Home component 1`] = `"<div class=\\"p-5\\"><h1 class=\\"h5\\">Diet tracker</h1><p>Diet tracker helps you keep track of your daily nutrition</p><p>Add your consumed carbohydrates, proteins and fats.</p><p>Stay motivated by reaching your week nutrition goals.</p></div>"`;
